web/elm/src/SideBar/Views.elm

```diff
@@ -67,6 +68,11 @@ type alias Pipeline =
     , background : Styles.Background
     , href : String
     , domID : DomID
+    , favIcon :
```

Comment:
```suggestion
    , starIcon :
```
Doesn't have to be `starIcon`, it could even be `favoriteIcon`, but I feel weird overlapping with the well-known web term: https://en.wikipedia.org/wiki/Favicon.